editoast/src/fixtures.rs

```diff
@@ -465,14 +465,15 @@ pub mod tests {
     }
 
     async fn make_small_infra(db_pool: Arc<DbConnectionPool>) -> Infra {
+        let conn = &mut db_pool.get().await.unwrap();
         let railjson: RailJson = serde_json::from_str(include_str!(
             "../../tests/data/infras/small_infra/infra.json"
         ))
         .unwrap();
         Infra::changeset()
             .name("small_infra".to_owned())
             .last_railjson_version()
-            .persist(railjson, db_pool)
+            .persist(railjson, conn)
```

Comment:
`conn` should probably be inlined here, to drop the connection as soon as possible.

```suggestion
            .persist(railjson, db_pool.get().await.unwrap().deref_mut())
```

Reply:
Done.